persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxGenerator.java

```diff
@@ -1048,17 +1053,20 @@ private static ModuleMemberDeclarationNode createUpdateRecord(Entity entity) {
 
     private static String getEntityNameConstant(String entityName) {
         StringBuilder outputString = new StringBuilder();
-        String[] splitedStrings = entityName.split(REGEX_FOR_SPLIT_BY_CAPITOL_LETTER);
+        String[] splitedStrings = stripEscapeCharacter(entityName).split(REGEX_FOR_SPLIT_BY_CAPITOL_LETTER);
         for (String splitedString : splitedStrings) {
             if (outputString.length() != 0) {
                 outputString.append(UNDERSCORE);
             }
             outputString.append(splitedString.toUpperCase(Locale.ENGLISH));
         }
+        if (entityName.startsWith(SINGLE_QUOTE)) {
+            return SINGLE_QUOTE + outputString.toString();
+        }
         return outputString.toString();
     }
 
-    private static String stripEscapeCharacter(String fieldName) {
-        return fieldName.startsWith("'") ? fieldName.substring(1) : fieldName;
+    public static String stripEscapeCharacter(String fieldName) {
```

Comment:
This method seems to be used only in this file. Do we have any reason to make it `public'?

Reply:
This was initially changed in order to use this in SQL script generation, but as it was not used in the final code, this was corrected in [50c4917](https://github.com/ballerina-platform/persist-tools/pull/137/commits/50c491782e9456a22a362eafd995ed8372ae16b4)